requirements.txt

```diff
@@ -0,0 +1 @@
+pyyaml
```

Comment:
Just a note for myself: pyyaml was already covered by our license compliance stuff for cve-bin-tool so I don't need to file new paperwork for this one.